src/java/org/apache/poi/common/usermodel/fonts/FontGroup.java

```diff
@@ -110,7 +116,7 @@ public FontGroup getFontGroup( ) {
      * @param runText the text which font groups are to be analyzed
      * @return the FontGroup
      */
-    public static List<FontGroupRange> getFontGroupRanges(final String runText) {
+    public static List<FontGroupRange> getFontGroupRanges(@Nullable final String runText) {
```

Comment:
I think the intent is for people to provide a non-null value. However, the function can handle `null` since it has an explicit null check. Leaving the `@Nullable` off and having the null check triggers an Intellij inspection. I figure adding `@Nullable` is probably an OK choice to 1. resolve the inspection 2. allow tools using the nullness annotations to not complain about passing something that _might_ be null.

I think a lot of parameters will end up being marked as `@Nullable`